server/src/main/kotlin/org/elaastic/player/results/learner/LearnerResultsModelFactory.kt

```diff
@@ -1,18 +1,61 @@
 package org.elaastic.player.results.learner
 
-import org.elaastic.player.explanations.ExplanationDataFactory
+import org.elaastic.activity.response.Response
 import org.elaastic.material.instructional.question.ExclusiveChoiceSpecification
 import org.elaastic.material.instructional.question.MultipleChoiceSpecification
+import org.elaastic.material.instructional.question.QuestionType
 import org.elaastic.material.instructional.statement.Statement
-import org.elaastic.activity.response.Response
+import org.elaastic.player.explanations.ExplanationDataFactory
 
 object LearnerResultsModelFactory {
 
-    fun buildOpenResult(responseFirstTry: Response?,
```

Comment:
I have no preference at all between `firstTry` or `firstAttempt`. But I do prefer to have a single terminology.
If I'm correct, the term `attempt` was used before. You should continue with it, or replace previous occurences.

Reply:
This is not a blocker. Something to have in mind, but that can be handled later...